nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/orient/HelmUploadHandler.java

```diff
@@ -62,12 +62,12 @@
 
   private final HelmAttributeParser helmPackageParser;
 
-
   @Inject
-  public HelmUploadHandler(final ContentPermissionChecker contentPermissionChecker,
-                           final HelmAttributeParser helmPackageParser,
-                           @Named("simple") final VariableResolverAdapter variableResolverAdapter,
-                           final Set<UploadDefinitionExtension> uploadDefinitionExtensions)
+  public HelmUploadHandler(
```

Comment:
Minor: redundant refactoring. Put "VCS only" checkbox in the refactoring menu.

Reply:
fixed 88bac16ecc93e6c7eb6f3fde288232ef8ea0468c